### PR TITLE
Update e2e test encryption and pws

### PR DESCRIPTION
**Pull request by Addison-Stavlo**
Related to #

## Proposed Changes

* updating e2e encryption file and tc config.

## Why are these changes being made?

* security

## Testing Instructions

* we will need E2Es ton this PR to pass once updating the test user passwords.

## Pre-merge Checklist

- [ ] Has the general commit checklist been followed? (PCYsg-hS-p2)
- [ ] [Have you written new tests](https://wpcalypso.wordpress.com/devdocs/docs/testing/index.md) for your changes?
- [ ] Have you tested the feature in Simple (P9HQHe-k8-p2), Atomic (P9HQHe-jW-p2), and self-hosted Jetpack sites (PCYsg-g6b-p2)?
- [ ] Have you checked for TypeScript, React or other console errors?
- [ ] Have you used memoizing on expensive computations? More info in [Memoizing with create-selector](https://github.com/Automattic/wp-calypso/blob/trunk/packages/state-utils/src/create-selector/README.md) and [Using memoizing selectors](https://react-redux.js.org/api/hooks#using-memoizing-selectors) and [Our Approach to Data](https://github.com/Automattic/wp-calypso/blob/trunk/docs/our-approach-to-data.md)
- [ ] Have we added the "[Status] String Freeze" label as soon as any new strings were ready for translation (p4TIVU-5Jq-p2)?
- [ ] For changes affecting Jetpack: Have we added the "[Status] Needs Privacy Updates" label if this pull request changes what data or activity we track or use (p4TIVU-aUh-p2)?

### CODE REVIEW

File: .teamcity/settings.kts [hunk 1]
```diff
@@ -78,10 +78,10 @@ project {
 		param("teamcity.git.fetchAllHeads", "true")
 
 		// e2e config decryption key references. See PCYsg-vnR-p2 for more info.
-		password("E2E_SECRETS_ENCRYPTION_KEY_DEC_1_23" , "credentialsJSON:1d6bc37e-2243-4fcb-b202-f7cc35e748da", display = ParameterDisplay.HIDDEN)
 		password("E2E_SECRETS_ENCRYPTION_KEY_JUN_27_24" , "credentialsJSON:6472fa88-cddb-4274-87c0-c935887e61eb", display = ParameterDisplay.HIDDEN)
```
Comment (tyxla) [inline .teamcity/settings.kts:81]: Technically we should be able to remove `E2E_SECRETS_ENCRYPTION_KEY_JUN_27_24` too, doesn't seem in use after this PR. Fine to be removed separately too.

**Review by tyxla — APPROVED** [review]
Haven't tested the decryption, but otherwise looks good to me 👍

**Comment by Addison-Stavlo** [comment]
> Haven't tested the decryption

Noting we also tested and confirmed this works - p1724866188762409/1724677645.233919-slack-C02DQP0FP

**Comment by tyxla** [comment]
Yup, thanks for nudging me to test it @Addison-Stavlo 🙌 - decryption worked correctly 👍

**Pull request merged**